src/App.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div id="q-app">
+    <router-view />
+  </div>
+</template>
+<script>
+import { mapGetters, mapActions } from 'vuex'
+import CategoriesAPI from 'src/services/categories/categories-api'
+
+export default {
+  name: "App",
+  mounted() {
```

Comment:
Em funções, for, if etc... colocamos espaço antes do parenteses e depois.
Exemplos:
```js
mounted () {}
if () {}
for () {}
```

Reply:
Normalmente chamamos a API no created tb